packages/core/docs/guide/getters.md

```diff
@@ -0,0 +1,171 @@
+# Getters
+
+## What are getters?
+
+Getters are pure functions which allows you to fetch response from integrated ecommerce platform. They return an agnostic or primitive type. All getters are grouped into domains they are associated with e.g. `cartGetters` and named according to values they return e.g. `getCartTotalItems`. 
```

Comment:
1. getters are not fetching anything, they're used to 
- extract subproperties from complex objects like products and categories
- deliver agnostic data formats for the UI
2. it's worth writing this in simpler language eg "All getters are grouped into domains they are associated with" => "Each composable has its own set of getters..."

Reply:
I changed this part and add another quiestions to be answered in this doc.